frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -121,7 +121,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
               {
-                (hasRole(currentUser, "ROLE_DRIVER") || hasRole(currentUser, "ROLE_ADMIN")) && (
+                isParticipant(currentUser) && !hasRole(currentUser, "ROLE_RIDER") && (
```

Comment:
I strongly prefer the first formulation.  

There is a better way to fix the mutation issues; I know because another team did it.

You simply need three tests: 

* One that checks that `Drivers Page` appears  when the user has `ROLE_DRIVER`
* One that checks that `Drivers Page` appears  when the user has `ROLE_ADMIN`
* One that checks that `Drivers Page` DOES NOT appear  when the user has `ROLE_RIDER`

Please revert this change, and then fix the mutation test issues by writing the three tests recommended.

In general, if you are making the code harder to read in order to pass mutation testing, you are doing something wrong.  Fix the tests, not the code.